contracts/tokens/SecurityToken.sol

```diff
@@ -53,10 +57,12 @@ contract SecurityToken is ISecurityToken, StandardToken, DetailedERC20 {
         uint256 _timestamp
     );
 
+    event LogUpdateTokenDetails(bytes32 _oldDetails, bytes32 _newDetails);
     event LogGranularityChanged(uint256 _oldGranularity, uint256 _newGranularity);
     event LogModuleRemoved(uint8 indexed _type, address _module, uint256 _timestamp);
     event LogModuleBudgetChanged(uint8 indexed _moduleType, address _module, uint256 _budget);
     event Mint(address indexed to, uint256 amount);
+    event Burn(address indexed _burner, uint256 _value);
```

Comment:
Can we change event names to:
Minted
Burnt
this keeps naming consistent (past tense) and differentiates a bit from the same function names.

Reply:
Maybe also move these events into IST20.sol?